### ISSUE TITLE
Hogs CPU and never finishes

### ISSUE THREAD

**Issue by norbusan** [swick/mozilla-gnome-keyring#46]
Hi
I am running firefox latest (50.0) with this plugin, and on sync it hogs up CPU for hours and hours, in fact it is the gnome-keyring-daemon that takes nearly 100% of the CPU. I tried this now on 2 of my computers and after logging into my firefox account and starting sync, CPU goes up on g-k-d to close to100, and remains there even after 24h ...

I would really like to see my passwords moved to the g-k store, but as it is now it is unfortunately unusable.

All the best

Norbert

Debian/sid, firefox 50.0, gnome-keyring-integration 0.12

**Comment by miles969** [comment]
hi!

this happens to me sometimes too... it seems that the chosen keyring gets updated when firefox is syncing, but it looks like all passwords are overwriting eachother. it looked like it was stuck in an update/delete loop while the sync was running.

regards
m

**Comment by norbusan** [comment]
yeah, afais every password is deleted and then recreated, one by one, and it seems that g-k-d is not very effectively implementing these updates.
so it might be "just another stupid Gnome thing", like so many others.
